assets/scripts/ajaxupload.js

```diff
@@ -509,7 +509,7 @@
             // "This page contains both secure and nonsecure items"
             // Anyway, it doesn't do any harm.
             iframe.setAttribute('id', id);
-            iframe.setAttribute('srcdoc', this._settings.action);
+            // iframe.setAttribute('srcdoc', this._settings.action);
```

Comment:
You can remove this line, no? If you only comment it, the reader won't know why it's commented (without checking git blame).

Reply:
Maybe comments temporary ? The issue was really strange.

But clearlky : all upload question type must be rewritten to use real ajax

Reply:
I am following up with patrick on why that line was added. There are some stackoverflow posts which recommend that. Maybe the chrome change that produced that was rolled back after?

We need to be able to reproduce the old bug (the one from couple of weeks ago).

I believe, we need to retest the initial bug, or said in other words identify the key browsers (and specific possible engine variations) where to test this.

Reply:
> You can remove this line, no?

Yes, was a work in progress. I asked Sushma to help me test it

Reply:
:+1: